content/zh/docs/application-store/external-apps/deploy-clickhouse.md

```diff
@@ -14,6 +14,7 @@ weight: 14340
 
 - 请确保[已启用 OpenPitrix 系统](../../../pluggable-components/app-store/)。
 - 您需要创建一个企业空间、一个项目和一个用户帐户 (`project-regular`) 供本教程操作使用。该帐户需要是平台普通用户，并邀请至项目中赋予 `operator` 角色作为项目操作员。本教程中，请以 `project-regular` 身份登录控制台，在企业空间 `demo-workspace` 中的 `demo-project` 项目中进行操作。有关更多信息，请参见[创建企业空间、项目、帐户和角色](../../../quick-start/create-workspace-and-project/)。
+- 请确保 KubeSphere 集群[已开启外网访问](../../../project-administration/project-gateway/)。
```

Comment:
The EN sentence has more information than the ZH sentence here. Please make sure they convey the same message.

Reply:
done

Reply:
Please use the ZH translation as shown in the screenshot. Besides, would it be better to say `KubeSphere 项目` than `KubeSphere 集群` since the document you referenced here talks about project administration?
![Xnip2021-06-18_18-00-51](https://user-images.githubusercontent.com/75110798/122544573-882c3080-d05f-11eb-82e3-78f5d7598cab.png)